implementations/rust/daemon/src/cli.rs

```diff
@@ -241,19 +257,23 @@ impl FromStr for VaultKind {
 pub enum ChannelRole {
     /// The Initiator role expects a channel responder address and a public key to use in order to
     /// communicate with the Responder end of the channel.
-    Initiator,
+    Source,
     /// The Responder role will create a channel responder, and will instruct the program to print
```

Comment:
If we're going to commit to using these names in code (which I think @mrinalwadhwa should approve) then can you also update the comments to reflect the change?

Reply:
done...